index.js

```diff
@@ -37,27 +38,42 @@ const format = (markup, url) => {
 
   const api = [];
 
-  console.log(`\n${chalk.bold(title)}\n`);
-  console.log(wrap(description));
-  console.log(`\n${usage}\n`);
-
-  $('#wikiArticle dl dt').each((index, element) => {
-    const $element = $(element);
-    const term = $element.text();
-    const definition = $element
-      .next('dd')
-      .text()
-      .replace(new RegExp(term, 'gim'), chalk.bold(term));
-
-    api.push({
-      term: chalk.bold(term),
-      definition
+  console.log(`\n${chalk.bold(title)}`);
+  console.log(`\n${wrap(description)}\n`);
+
+  if (/[a-z]/.test(usage)) {
+    console.log(`${usage}\n`);
+  }
+
+  $('#wikiArticle dl dt')
+    .has('code')
+    .each((index, element) => {
+      const $element = $(element);
+      const term = $element.text();
+      const definition = $element
+        .next('dd')
+        .text()
+        .replace(new RegExp(term, 'gim'), chalk.bold(term));
+
+      api.push({
+        term: chalk.bold(term),
+        definition
+      });
+
+      api.push({term: '', definition: ''});
     });
 
-    api.push({term: '', definition: ''});
+  const cleanApi = api.map(t => {
```

Comment:
@greg-js I usually try to avoid single letters and abbreviations and always go with the more verbose option. So, can you please change `t` to something more descriptive?